README.md

```diff
@@ -120,48 +129,175 @@ list, I am an expert in none of these technologies).
+   framework for Elixir) are great choices for Rubyist as Elixir was written by
+   a former Rails core contributor.
 
 ### Introduce yourself to data-science
 
-1. I am currently following the
-   [Data Analyst Nanodegree on Udacity](https://www.udacity.com/course/data-analyst-nanodegree--nd002).
-   It have been great so far!
+1. I am currently following the [Data Analyst Nanodegree on Udacity]. It have
+   been great so far!
 
 ### Focus on front-end
 
-1. Improve your CSS skills
-2. Learn a trendy JS library (React or Vue.js for example). Good resources
-   include Wes Bos stuffs or EggHead tutorial. I personnaly have a EggHead
-   subscription as it allows to quickly get introduced to new topics before
-   going deeper by yourself if it catches your interest.
+1. Learn how to structure properly your vanilla CSS with [BEM]: for reusable
+   components & a real naming convention for your next big projects.
+1. Then, show your skills to the entire world on [Codepen] by sharing some
+   [next level Japanese Sushi Toggle Button] ! :yum:
+1. Of course, your Codepen profile can be written in something a bit more
+   advanced if you wish. Why not [Stylus] (some cool features and ponctuation
+   free) and [Haml] (_a bit_ more readable HTML) ?
+1. Bootstrap is nice but you want to discover other CSS frameworks ? There are
+   plenty of them, like [Materialize], [Bulma] or [Carbon Design System].
+1. Want to know what Bootstrap does behind the scenes in it's 4th version ? Wes
+   has a free [Flexbox] lesson just for you ! You'll see that it's helps a lot
+   for some homemade responsive Webdesign.
+1. CSS Grid is another cool feature released not so long ago. We don't have any
+   goods links here but feel free to pull request one ! :smirk:
+1. Tell me something, do you like the animations of this [studio] ? It's using
+   [Greensock], part of it is free and it allows you to make some awesome
+   design effects. [Petr Tichy] has some free lessons on it !
 
 ### Become a mobile developer
 
-1. Learn React Native (and plug it to a Rails API) to build native applications
-   on mobile
+1. Learn a trendy JS library (ReactJS or Vue.js for example). One of the best
+   website for updated lessons is [egghead]. I do personnaly have a subscription
+   as it allows to quickly get introduced to new topics before going deeper by
+   yourself if it catches your interest (it is sometimes written directly by
+   the founders too) !
+1. Get hyped of the possibilites of React Native by this [presentation] of
+    Sébastien Saunier.
+1. Learn a bit more of how to link React to your Rails app with these pair of
+   [slides].
+1. Then, if you're conviced and want to create native mobile apps for Android &
+   iPhone, feel free to buy some [Full Stack courses] from Stephen Grider !
+   :tada: They aren't very expensive, 10$ each.
+1. If you're not really into JavaScript, you can try some new things like
```

Comment:
Same, why in the mobile section

Reply:
It's in the "mobile" (currently renamed to `Discover a front end framework`) because his courses are pretty much all the requirements you need to be able to go from 0 -> Some ReactJS/Native projects done as presented during [this](http://slides.com/fcatuhe/react-on-rails#/7/2) Bordeaux Ruby Apéro.

Reply:
Ok, why not